### PR TITLE
Olha Mazurenko

### PR DESCRIPTION
**Pull request by OlhaMazurenko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+<body>
+  
+  <script src="./main.js"></script>
+</body>
+</html>
```
Comment (gestap) [inline index.html]: the document must end with one empty line

File: main.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+document.addEventListener('DOMContentLoaded', () => {
+let n = new Numberdrome();
```
Comment (gestap) [inline main.js]:
```suggestion
  let n = new Numberdrome();
```

File: main.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  addNumber(n) {
+    this.num.push(n);
+  }
+
+  removeNumber(n) {
+    this.num.splice(this.num.indexOf(n), 1);
+  }
+
+  sum() {
+    return this.num.reduce(((x, y) => x + y),0)
```
Comment (gestap) [inline main.js]:
```suggestion
    return this.num.reduce(((x, y) => x + y), 0)
```

File: main.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  min() {
+    try {
+      if (!this.num.length) {
+        throw 'error!'
+      }
+    return Math.min(...this.num);
+    } catch (e) {
+      console.log(e);
```
Comment (gestap) [inline main.js]: you need to `throw an error` instead return message

File: main.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  max() {
+    try {
+      if (!this.num.length) {
+        throw  'error!'
+      }
+    return Math.max(...this.num);
+    } catch (e) {
+      console.log(e);
```
Comment (gestap) [inline main.js]: you need to `throw an error` instead return message

File: main.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+}
```
Comment (gestap) [inline main.js]: the document must end with one empty line

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OlhaMazurenko) [reply index.html]: Done

File: main.js (hunk 2 above)
Comment (OlhaMazurenko) [reply main.js]: Done

File: main.js (hunk 3 above)
Comment (OlhaMazurenko) [reply main.js]: Done

File: main.js (hunk 4 above)
Comment (OlhaMazurenko) [reply main.js]: Done

File: main.js (hunk 5 above)
Comment (OlhaMazurenko) [reply main.js]: Done

File: main.js (hunk 6 above)
Comment (OlhaMazurenko) [reply main.js]: Done

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**